pandas/tests/io/json/test_pandas.py

```diff
@@ -1750,3 +1750,23 @@ def test_readjson_bool_series(self):
         result = read_json("[true, true, false]", typ="series")
         expected = Series([True, True, False])
         tm.assert_series_equal(result, expected)
+
+    def test_to_json_multiindex_escape(self):
+        # GH 15273
+        df = DataFrame(
+            True,
+            index=pd.date_range("2017-01-20", "2017-01-23"),
+            columns=["foo", "bar"],
+        ).stack()
+        result = df.to_json()
+        expected = (
+            "{\"(Timestamp('2017-01-20 00:00:00'), 'foo')\":true,"
```

Comment:
woa, this is the expected?

Reply:
I believe so; yes.

The result is similar to https://github.com/pandas-dev/pandas/issues/15273#issue-204313476 except I think it's correct that we retain the timestamp like object instead of converting to epoch timestamp